src/integrationTest/java/com/mongodb/hibernate/query/select/SimpleSelectQueryIntegrationTests.java

```diff
@@ -321,6 +322,15 @@ void testNullParameterNotSupported() {
                     FeatureNotSupportedException.class,
                     "TODO-HIBERNATE-74 https://jira.mongodb.org/browse/HIBERNATE-74");
         }
+
+        @Test
+        void testQueryPlanCacheIsNotForbidden() {
```

Comment:
[optional]

Let's use "IsSupported" or "IsAllowed" instead of "IsNotForbidden": 

```suggestion
        void testQueryPlanCacheIsSupported() {
```